portal/App.js

```diff
@@ -77,7 +77,8 @@ import BusinessMarketMd from './../packages/theme/business-market/BUSINESS-MARKE
 
 // App context
 const AppContext = React.createContext({
-  sideBarActive: false
+  blackAndWhiteActive: false,
+  sideBarActive: false,
```

Comment:
Shouldn't we make this an enum? ColorMode.Default, ColorMode.Greyscale, ColorMode.Protanopia, etc?

Reply:
Only needed if we have multiple eye-disorders, i think...

Reply:
Yes, when we add more modes that will be great: https://reactjs.org/docs/typechecking-with-proptypes.html